tests/lib/rules/comma-spacing.js

```diff
@@ -61,6 +61,7 @@ ruleTester.run("comma-spacing", rule, {
         "['  ,  ']",
         { code: "[`  ,  `]", parserOptions: { ecmaVersion: 6 } },
         { code: "`${[1, 2]}`", parserOptions: { ecmaVersion: 6 } },
+        { code: "fn(a, b,)", parserOptions: { ecmaVersion: 2018 } }, // #11295
```

Comment:
Would it possible to also include tests with an arrow function, function expression, and function declaration, just to document the expected behavior? Thanks!